src/conv-core/debug-conv.h

```diff
@@ -10,7 +10,7 @@
 /*If you are incrementing these numbers, you also need to increment MAJOR and MINOR
 variables in ParDebug.java(in ccs_tools) to match, otherwise CharmDebug won't work*/
 #define CHARMDEBUG_MAJOR   10
-#define CHARMDEBUG_MINOR   10
+#define CHARMDEBUG_MINOR   11
```

Comment:
Was this intentional? When is it required to increment these numbers? 

Also, note the comment about incrementing ccs_tools.

Reply:
It's mentioned in charm.h at line 316. Although the previous changes for group broadcasts were never merged from this PR: https://github.com/UIUC-PPL/ccs_tools/pull/2 so I'll have to address that as well.

Reply:
Line 316 reads "/*** WARNING!!!! The following enum is linked to charmdebug finals in MsgInfo.java.
 *   Make sure the two remain synchronized if changing this one." 

So, everytime the message type enum is updated, is it required to increment the CHARMDEBUG version? Why is only CHARMDEBUG_MINOR updated?